### ISSUE TITLE
Youtube Embed iframe has poor focus indicators

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1051]
What is Health Equity	Keyboard	[A11Y - Health Equity Tracker]: Faint visual indicator for items in What is Health Equity page when user navigates using tab navigation_Web_1.2	"ENVIRONMENT:
Platform: macOS 11.3
Browser: Chrome Version 90.0.4430.212 (Official Build) (x86_64)
STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/whatishealthequity
2). Navigate to what is health equity tab and click on it
3). Navigate through what is health equity page using tab key
4). Observe the visual indicator on the page. 
EXPECTED RESULT:
Element having keyboard focus should have a visual indicator demarcating the same.
OBSERVED RESULT:
Some elements on what is health equity page have a very faint visual indicator when user navigates using tab key. It is very difficult for a user to see the visual indicator and hence the keyboard focus for those elements.
Replication video attached"	https://drive.google.com/file/d/1-Xi1UZ9aAo73e2CPgrdhLA4qHRuYO7rF/view?usp=sharing

**Comment by benhammondmusic** [comment]
The styling for the focus ring within this embedded Youtube iframe is set by Youtube; I'm not sure we have any way to override this and make a matching / more visible focus selector.
